src/gurt/heap.c

```diff
@@ -497,7 +499,8 @@ d_binheap_insert(struct d_binheap *h, struct d_binheap_node *e)
 	if (h->d_bh_ops->hop_enter) {
 		rc = h->d_bh_ops->hop_enter(h, e);
 		if (rc != 0) {
-			D_ERROR("d_bh_ops->hop_enter failed, rc: %d.\n", rc);
+			D_ERROR("d_bh_ops->hop_enter failed, " DF_RC "\n",
```

Comment:
is it worth adding () here?
D_ERROR("d_bh_ops->hop_enter() failed, " DF_RC "\n",

Reply:
This is consistent with the rest of the file, but it's a small file.  Will change.